tests/admin_widgets/tests.py

```diff
@@ -687,6 +687,23 @@ def value_omitted_from_data(self, data, files, name):
         wrapper = widgets.RelatedFieldWidgetWrapper(widget, rel, widget_admin_site)
         self.assertIs(wrapper.value_omitted_from_data({}, {}, 'band'), False)
 
+    def test_widget_is_hidden(self):
+        rel = Album._meta.get_field('band').remote_field
+        widget = forms.HiddenInput()
+        widget.choices = ()
+        wrapper = widgets.RelatedFieldWidgetWrapper(widget, rel, widget_admin_site)
+        self.assertTrue(wrapper.is_hidden)
```

Comment:
Use `self.assertIs(wrapper.is_hidden, True)` since assertTrue passes for bool(value) is True.

Reply:
Thanks, that's much safer.